test/tbb/test_limiter_node.cpp

```diff
@@ -623,5 +623,5 @@ TEST_CASE("Test correct node deallocation while using small_object_pool") {
     graph.wait_for_all();
 
     tbb::task_scheduler_handle handle = tbb::task_scheduler_handle::get();
-    REQUIRE_NOTHROW( tbb::finalize( handle, std::nothrow ) );
```

Comment:
@vlserov could you please clarify that the compiler complains about, and why this change fixes the problem? As, at least to me, changes seems a bit of irrational :)

Reply:
Since `tbb::finalize` is marked `noexcept`, MSVC 2015 compiler assumes that some part of macro `REQUIRE_NOTHROW` cannot be reached. All tests are assembled with `/EHsc`(Standard C++ exception handling model), Structured Exceptions (SEH) cannot be handled by C++ `try..catch` block without `/EHa` flag.
`REQUIRE_NOTHROW` can be removed.
Another possible solution is using `/EHa` flag for test_limiter_node.